crates/bevy_asset/src/io/mod.rs

```diff
@@ -130,7 +130,10 @@ where
 /// API, where asset bytes and asset metadata bytes are both stored and accessible for a given
 /// `path`. This trait is not object safe, if needed use a dyn [`ErasedAssetReader`] instead.
 ///
-/// Also see [`AssetWriter`].
+/// This trait defines asset-agnostic mechanisms to read bytes from a storage system.
```

Comment:
The first and second paragraphs seem like they should be combined somehow? It feels a bit duplicated to me.

Reply:
I don't immediately see a good way to do this, so I'm going to merge as is <3